boards/st/nucleo_h743zi/nucleo_h743zi.dts

```diff
@@ -160,7 +160,7 @@ zephyr_udc0: &usbotg_fs {
 };
 
 &adc3 {
-	pinctrl-0 = <&adc3_inp5_pf3>;
+	pinctrl-0 = <&adc3_inp5_pf3 &adc3_inp0_pc2_c>;
```

Comment:
PC2_C is a special pin that may require a call to the LL_SYSCFG_OpenAnalogSwitch function if PC2 come to be used for something else (to separate the 2 pads). Is there a need to add this specific pin?